client-js/app/scripts/activities.ts

```diff
@@ -370,17 +370,16 @@ export class ManagerActivity implements Activity {
               private api: manager.ManagerApi) {}
 
   async start(): Promise<ActivityState> {
-    var conns = await this.api.connections();
-    var connsWithUrl = conns.map((c) => {
-      var x = <any> c;
+    var connections = await this.api.connections();
```

Comment:
Made some changes to dedupe this. The connection info is still loaded by the activity, but goes through a `setConnections` method instead of just setting the property so that the urls get updated in one place. That way the view doesn't render empty before the first time connection info is loaded.